ui/package.json

```diff
@@ -181,16 +182,16 @@
     "react-loadable": "^5.5.0",
     "react-markdown": "^4.0.3",
     "react-monaco-editor": "^0.33.0",
-    "react-redux": "^5.1.2",
-    "react-router": "^3.0.2",
-    "react-router-redux": "^4.0.8",
+    "react-redux": "^7.2.0",
+    "react-router-dom": "^5.2.0",
     "react-scrollbars-custom": "^4.0.0-alpha.8",
     "react-virtualized": "^9.18.5",
     "redux": "^4.0.0",
     "redux-auth-wrapper": "^1.0.0",
     "redux-thunk": "^2.3.0",
     "reselect": "^4.0.0",
     "rome": "^2.1.22",
+    "seamless-immutable": "^7.1.3",
```

Comment:
what did this solve?

Reply:
It’s seamless

Reply:
its a dependency to for one of the upgrades and to get yarn to stop yelling at me.